config/locales/blacklight.en.yml

```diff
@@ -1,3 +1,6 @@
 en:
   blacklight:
-    application_name: 'Blacklight'
\ No newline at end of file
+    application_name: 'Blacklight'
+    search:
+      fields:
+        license: 'Rights'
```

Comment:
@seanaery Not sure why this is needed.  Not saying it's wrong, just not sure why it's part of this PR.

Reply:
@coblej Good eye. The "license" field gets tacked on to the bottom of the metadata section via this partial in the Hyrax codebase:
https://github.com/samvera/hyrax/blob/master/app/views/hyrax/base/_attributes.html.erb#L11

That partial gets its label from blacklight.en.yml. I wasn't quite sure if it was more recommended to replicate that partial in RDR to remove that display in favor of treating license like the other metadata fields in attribute_rows, versus just specify the label here.

Reply:
@seanaery I think you made the right choice.  Thanks for the explanation.